### ISSUE TITLE
Create Team + Create Namespace is a bit confusing

### ISSUE THREAD

**Issue by jstrachan** [hawtio/hawtio-kubernetes#112]
see: https://www.dropbox.com/s/33mxm3dzuvoxr0s/Screenshot%202016-06-03%2010.39.49.png?dl=0

Maybe lets rename the second button to be consistent? 

When showing the regular namespaces we maybe hide those lower buttons?

**Comment by gashcrumb** [comment]
'create namespace' still just creates a namespace, it's the old dialog.
The create team wizard will do that job also, but also has the extra steps
to prompt for deploying CD pipeline etc. and creates the
fabric8-environments config map.  Could fix the wizard to prompt for that
instead so the user can choose to go back to the main page or create the
config map and continue with the wizard?

**Comment by jstrachan** [comment]
@gashcrumb aha sorry, I'd missed that!

So maybe we need 2 buttons; or unify them in a Create wizard? I guess creating an Environment only makes sense within a Team though; so maybe there's a Create Environment button on the Team/Project page?

**Comment by gashcrumb** [comment]
Yeah, so create team just creates the config map, but doesn't create any
environments.  So we have the config map for detecting a team and hopefully
will have CD pipeline deploying, then going into the team prompts to create
a project and the environments get created from there.
